code/modules/awaymissions/mission_code/snowdin.dm

```diff
@@ -212,21 +212,22 @@
 				if(ishuman(L))//are they a carbon?
 					var/list/plasma_parts = list()//a list that'll store the limbs of our victim
 					var/mob/living/carbon/human/PP = L
-					if(istype(PP.dna.species, /datum/species/plasmaman))//is the species of the mob a podperson?
+					if(istype(PP.dna.species, /datum/species/plasmaman))
 						return //don't bother with plasmamen here
 
 					for(var/BP in PP.bodyparts) //getting the victim's current body parts
 						var/obj/item/bodypart/NN = BP
 						if(NN.status == BODYPART_ORGANIC || NN.species_id != "plasmaman") //getting every organic, non-plasmaman limb (augments/androids are immune to this)
 							plasma_parts += NN //adding the limbs we got to the above-mentioned list
 
-					if(prob(35) && plasma_parts) //checking if the delay is over & if the victim actually has any parts to nom
+					if(prob(35)) //checking if the delay is over & if the victim actually has any parts to nom
 						PP.adjustToxLoss(15)
 						PP.adjustFireLoss(25)
-						var/obj/item/bodypart/NB = pick(plasma_parts) //using the above-mentioned list to get a choice of limbs for dismember() to use
-						NB.species_id = "plasmaman"//change the species_id of the limb to that of a plasmaman
-						PP.visible_message("<span class='warning'>[L] screams in pain as their [NB] melts down to the bone!</span>", \
-										  "<span class='userdanger'>You scream out in pain as your [NB] melts down to the bone, leaving an eerie plasma-like glow where flesh used to be!</span>")
+						if(plasma_parts.len)
+							var/obj/item/bodypart/NB = pick(plasma_parts) //using the above-mentioned list to get a choice of limbs for dismember() to use
+							NB.species_id = "plasmaman"//change the species_id of the limb to that of a plasmaman
+							PP.visible_message("<span class='warning'>[L] screams in pain as their [NB] melts down to the bone!</span>", \
+											  "<span class='userdanger'>You scream out in pain as your [NB] melts down to the bone, leaving an eerie plasma-like glow where flesh used to be!</span>")
```

Comment:
im not 100% sure this works as dream daemon keeps crashing on me when trying to start but it 'seems' like it should?

Reply:
you should set ```no_update``` on the limb, so that it will stay plasma.